Specs/Core/CesiumTerrainProviderSpec.js

```diff
@@ -265,19 +264,21 @@ describe("Core/CesiumTerrainProvider", function () {
       url: "made/up/url",
     });
 
-    return pollToPromise(function () {
-      return provider.ready;
-    }).then(function () {
-      expect(provider.getLevelMaximumGeometricError(0)).toBeGreaterThan(0.0);
-      expect(provider.getLevelMaximumGeometricError(0)).toEqualEpsilon(
-        provider.getLevelMaximumGeometricError(1) * 2.0,
-        CesiumMath.EPSILON10
-      );
-      expect(provider.getLevelMaximumGeometricError(1)).toEqualEpsilon(
-        provider.getLevelMaximumGeometricError(2) * 2.0,
-        CesiumMath.EPSILON10
-      );
-    });
+    return provider.readyPromise
+      .then(function () {
+        expect(provider.getLevelMaximumGeometricError(0)).toBeGreaterThan(0.0);
+        expect(provider.getLevelMaximumGeometricError(0)).toEqualEpsilon(
+          provider.getLevelMaximumGeometricError(1) * 2.0,
+          CesiumMath.EPSILON10
+        );
+        expect(provider.getLevelMaximumGeometricError(1)).toEqualEpsilon(
+          provider.getLevelMaximumGeometricError(2) * 2.0,
+          CesiumMath.EPSILON10
+        );
+      })
+      .catch((e) => {
+        console.log(e);
```

Comment:
If you're seeing an error here, it may be because the `readyPromise` in the previous spec isn't being waited on, and throwing something during spec cleanup.

Reply:
Yeah, nearly all my testing woes were due to issues like this. Thanks for the tip!